homeworks/olha.tymchyshyn_olyatmchshn/PokemonProject/scripts/script.js

```diff
@@ -0,0 +1,86 @@
+const EMAIL_INPUT_ID = 'email';
+const PASSWORD_INPUT_ID = 'password';
+const NOT_A_ROBOT_CHECKBOX_ID = 'robotcheck';
+const SUBMIT_BUTTON_ID = 'submit-button';
+const ERRORS_CONTAINER_ID = 'errors-container';
+const RESULT_PAGE_PATH = '../welcome.html';
```

Comment:
Success login should lead to welcome page, please correct the path to html-file. 
<img width="889" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/4a0a31fe-7d02-4439-86a0-e7e328ae8e0f">